spec/factories/application_form.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_duplicate do
+      date_of_birth { '01-01-1996' }
+      postcode { 'W6 9BH' }
+      last_name { 'Thompson' }
+    end
+
```

Comment:
We already have a `duplicate_candidates` trait for `ApplicationForm`. Can we consolidate these?